src/ibex_bluesky_core/callbacks/fitting/fitting_utils.py

```diff
@@ -179,8 +179,8 @@ class Linear(Fit):
     def model(cls, *args: int) -> lmfit.Model:
         """Linear Model."""
 
-        def model(x: npt.NDArray[np.float64], m: float, c: float) -> npt.NDArray[np.float64]:
-            return m * x + c
+        def model(x: npt.NDArray[np.float64], c1: float, c0: float) -> npt.NDArray[np.float64]:
```

Comment:
I think the docs now need updating to match these updated parameter names?

Reply:
I also noticed the docs use `a` `b` `c` `d` for polynominal parameters but the code uses `c0`, `c1`, ...